JDP200503-18_Encja Group testy

tomysnatalia opened the pull request.

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
Merge branch 'master' into JDP200503-18-GroupTests

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

pplutap commented on src/main/java/com/kodilla/ecommercee/domain/Product.java:
```diff
@@ -2,15 +2,19 @@
     private String description;
     private double price;
     private String groupId;
+    private Group group;
```
Pole `group` nie jest potrzebne jeśli posiadamy `groupId` - o tym koncepcie (przypisywanie id zamiast całego obiektu) rozmawialiśmy w trakcie ostatniej rozmowy. Możemy temat poruszyć podobnie, było z tym kilka problemów

MarcinSchmechel replied:
Jestem za tym aby przypisywać cały obiekt - jeżeli ma być String to prośba o pomoc, jak to ma wyglądać.

pplutap commented on src/test/java/com/kodilla/ecommercee/repository/GroupRepositoryTestSuite.java:
```diff
@@ -0,0 +1,98 @@
+       // //When
+       // groupRepository.save(group1);
+       // //Then
+       //Long id = group1.getId();
```
Testy nie powinny być zakomentowane

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
git commit -am JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy

tomysnatalia committed:
JDP200503-18_Encja Group testy
